parsl/tests/test_staging/test_docs_1.py

```diff
@@ -12,6 +12,7 @@ def convert(inputs=[], outputs=[]):
 
 
 @pytest.mark.cleannet
+@pytest.mark.shared_fs
```

Comment:
this should probably be staging_required not shared_fs -- it's the job of the configured staging code to move these files to/from their worker locations, not any shared fs